pandas/tests/util/test_assert_almost_equal.py

```diff
@@ -80,6 +80,35 @@ def test_assert_almost_equal_numbers(a, b):
     _assert_almost_equal_both(a, b)
 
 
+@pytest.mark.parametrize(
+    "a,b",
+    [
+        (1.1, 1.1),
+        (1.1, 1.100001),
+        # check_less_precise=True allows 1.1 vs 1.1001
+        (1.1, 1.1001),
+    ],
+)
+def test_assert_almost_equal_numbers_less_precise(a, b):
+    _assert_almost_equal_both(a, b, check_less_precise=True)
+
+
+@pytest.mark.parametrize(
+    "a,b",
+    [
+        (1.1, 1.1),
+        (1.1, 1.100001),
+        (1.1, 1.1001),
+        # check_low_values allows for 0.1 vs 0.1001
+        (0.1, 0.1001),
+        # Testing this example, as per #13357
+        (0.000011, 0.000012),
+    ],
+)
+def test_assert_almost_equal_numbers_low_values(a, b):
+    _assert_almost_equal_both(a, b, check_less_precise=True, check_low_values=True)
```

Comment:
can you add a check that this assertion fails when it is suppose dto

Reply:
Done. Please check if the examples suffice. Thanks